Formula/radare2.rb

```diff
@@ -73,13 +86,22 @@ def install
     end
     system "make", "install"
 
+    resource("extras").stage do
+      ENV.append_path "PATH", bin
+      ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
+
+      system "./configure", "--prefix=#{prefix}"
+      system "make", "all"
+      system "make", "install"
+    end
+
     resource("bindings").stage do
       ENV.append_path "PATH", bin
       ENV.append_path "PKG_CONFIG_PATH", "#{lib}/pkgconfig"
 
       # Language versions.
       perl_version = `/usr/bin/perl -e 'printf "%vd", $^V;'`
-      lua_version = "5.1"
+      lua_version = `lua -e '_MAJ,_MIN=string.match(_VERSION,"^[^%d]+(%d+)%.(%d+).*$");io.write(table.concat({_MAJ,".",_MIN}))'`
```

Comment:
I can't imagine they'd remove it at this point FWIW. The `-v` flag has been present in 5.1, 5.2 & 5.3 so far, albeit with it being redirected to `stderr` rather than `stdout` in 5.1.